Gems/ROS2/Code/Source/ContactSensor/ROS2ContactSensorComponent.cpp

```diff
@@ -112,6 +119,11 @@ namespace ROS2
 
     void ROS2ContactSensorComponent::FrequencyTick()
     {
+        if (!IsComponentValid())
```

Comment:
Is there a reason why this check is preferred to just having the Component Deactivate?

In any case, I prefer to have it in one place (depending on solution, ROS2SensorComponent, or a global handler).